app/modules/sqlite3.c

```diff
@@ -31,7 +31,7 @@
 #define SQLITE_OMIT_PROGRESS_CALLBACK 1
 
 #include <c_stdlib.h>
```

Comment:
Just looked and there is still that one `c_` header - can it be swapped to `stdlib.h`? AFAIK SQLite is turned off completely so it shouldn't matter anyway - just a nit pick.

Reply:
D'oh, I must've missed that one.